setup.py

```diff
@@ -0,0 +1,38 @@
+import setuptools
+
+with open("README.md", "r") as fh:
+    long_description = fh.read()
+
+setuptools.setup(
+    name="metrixpp",
+    version="0.0.1a",
+    author="Stefan Strobel",
```

Comment:
is it your name? just checking that you have not left someone else's name from an example :)

Reply:
Yes, that's me ;)